modules/indexer/issues/indexer.go

```diff
@@ -99,6 +101,17 @@ func InitIssueIndexer(syncReindex bool) {
 			}
 			populate = !exist
 			holder.set(issueIndexer)
+		case "elasticsearch":
+			issueIndexer, err := NewElasticSearchIndexer(setting.Indexer.IssueConnStr, "gitea_issues")
```

Comment:
`gitea_issues` should be changeable from app.ini so that single elastic instance could be used for multiple Gitea instances

Reply:
done.